data/input/scenario_list.csv

```diff
@@ -1,2 +1,2 @@
-short_name,title,area_code,end_year,description,id,keep_compatible,heat_network_order,curve_file,heat_demand
-test_scen,Scenario-Tools Test Scenario,GM0363_amsterdam,2050,"""test scenario""",892503,False,energy_heat_burner_waste_mix energy_heat_burner_network_gas,2050_price_curves,
+short_name,title,area_code,end_year,description,id,keep_compatible,heat_network_order,curve_file,heat_demand
+test_hengelo,Hengelo test,GM0164_hengelo,2050,,,FALSE,,,
```

Comment:
Is this something we would like to commit?

Reply:
I restored the information in template CSVs to the inputs from master.